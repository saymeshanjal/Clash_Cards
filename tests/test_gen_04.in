10
14 4 1
10 24 1
2 40 1
16 40 4
14 35 2
14 1 5
6 11 3
10 2 5
2 43 4
2 15 4
