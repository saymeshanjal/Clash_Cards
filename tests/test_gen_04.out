1254 2178 0
124 128 0
1 1 0
416 9870 2584
140 328 2964
0 3432 0
1 19 0
0 252 0
0 2 0
0 2 0
